lxd/instance/drivers/driver_lxc.go

```diff
@@ -6382,6 +6382,16 @@ func (d *lxc) MigrateReceive(args instance.MigrateReceiveArgs) error {
 	}
 }
 
+// PostMigrateSend performs any required cleanup steps after an instance has been migrated to another member.
+func (d *lxc) PostMigrateSend() error {
+	err := d.postMigrateSendCommon(d)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Maybe it's a question of style/convention but I find this to read better.
```suggestion
	return d.postMigrateSendCommon(d)
```